openpype/settings/lib.py

```diff
@@ -172,7 +172,7 @@ def save_studio_settings(data):
     clear_metadata_from_settings(new_data)
 
     changes = calculate_changes(old_data, new_data)
-    modules_manager = ModulesManager(_system_settings=new_data)
+    modules_manager = ModulesManager(new_data)
```

Comment:
I had seen this change elsewhere, different PR - but I'm not sure why Github shows this as a change in this PR?

Reply:
It could be because I didn't update the branch yet